### PR TITLE
Test pull request

### PR DESCRIPTION
**Pull request by guschefi**
This is a test pull request

### CODE REVIEW

**Comment by wrang308** [comment]
Just two comments lol.

File: docs/index.html [hunk 1]
```diff
@@ -14,6 +14,8 @@
 			</div>
 			<div class="welcomeText">
 				<h2>Please select a JSON File</h2>
+				<!-- This is an awesome comment -->
```
Comment (wrang308) [inline docs/index.html:17]: is that a comment?

Comment (guschefi) [reply docs/index.html:17]: Actually, hard to believe but yeah!

**Review by omarfawal — APPROVED** [review]
Nice

**Pull request merged**